phase08/StudentGrades/StudentGrades/OutputPrinter.cs

```diff
@@ -0,0 +1,16 @@
+using DefaultNamespace;
+
+namespace project;
+
+public class OutputPrinter
+{
+    public void PrintTopStudents(List<Student> students)
+    {
+        using (var schoolContext = new SchoolDBContext())
+        {
+            schoolContext.Students.OrderByDescending(x => x.AverageGrade).Take(3).ToList().ForEach(student =>
+                Console.WriteLine
+                    ($"Name: {student.FirstName} {student.LastName} | Average Score: {student.AverageGrade}"));
```

Comment:
this method is not much general. override `ToString()` method instead.
**+** ordering and taking the first three element of students' list is not main functionality of this class. it should only print a list.